lib/v2/curius/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'https://curius.app/': {
```

Comment:
```suggestion
    'curius.app': {
```
You need to specify the root domain only